OracleDatabase/RAC/OracleDNSServer/README.md

```diff
@@ -0,0 +1,67 @@
+# Oracle DNS Server to resolve Oracle RAC IPs
+
+Sample container build files to facilitate installation, configuration, and environment setup for DevOps users.
+
+**IMPORTANT:** This image can be used to setup DNS server for RAC. You can skip this step if you already have a DNS server configure and which can be used for Oracle RAC. You should ensure that the DNS server container is up before starting RAC. This image is provided for test purposes only.
+
+## How to build and run
+You need to make sure that you have at least 350MB of space available for the container to create the files for RAC DNS server.

```

Comment:
If someone doesn't have 350MB available for this image, they're not going to have enough for RAC. This seems like a weird thing to call out.